python/tk_framework_alias/server/utils/qt.py

```diff
@@ -21,29 +22,35 @@
 try:
     from PySide2 import QtCore
 
-    if QtCore:
-        qt_app = QtCore.QCoreApplication.instance()
+    QtCore5 = QtCore
+    qt_app = QtCore.QCoreApplication.instance()
 except Exception:
     pass
 
 # Check if Qt was import successfully, if not try PySide6.
-if not QtCore or not qt_app:
+if not (QtCore and qt_app):
     if QtCore:
         # Reset QtCore if it was previously imported
         del QtCore
         QtCore = None
     try:
         from PySide6 import QtCore
 
-        if QtCore:
-            qt_app = QtCore.QCoreApplication.instance()
+        qt_app = QtCore.QCoreApplication.instance()
     except Exception:
         pass
 
-# Verify that QtCore module was imported and exists, and the qt app is accessible
-if not QtCore:
-    raise QtModuleNotFound("QtCore module not found. Failed to import PySide.")
-if not qt_app:
-    raise QtAppInstanceNotFound(
-        f"Qt App instance not found for Qt {QtCore.__version__}"
-    )
+# Verify that Qt was imported, and the qt app is accessible
+if not (QtCore or QtCore5):
```

Comment:
It's not clear to me why are you adding `QtCore5` variable. Maybe `QtCore` is enough at this point.

Reply:
I added the `QtCore5` variable to know specifically if we were able to import PySide2 successfully. This is to help debug PySide errors with Alias, and depending on the Alias version it will use PySide2 or PySide6.

This way, we can provide a more detailed error message for the specific error:

1. No PySide version could be import (not PySide6 and not PySide2)
2. We can import PySide6 but we could not get the running Qt6 App
3. We can import PySide2 but we could not get the running Qt5 App

That being said as I look the error handling over, I do not need the last `else` since we would never reach that case given the first `if` statement